WeakAuras/Prototypes.lua

```diff
@@ -2388,17 +2388,12 @@ WeakAuras.event_prototypes = {
         type = "string",
         init = "arg",
         enable = function(trigger)
-          return not WeakAuras.IsClassic() and trigger.subeventPrefix and (trigger.subeventPrefix:find("SPELL") or trigger.subeventPrefix == "RANGE" or trigger.subeventPrefix:find("DAMAGE"))
+          return trigger.subeventPrefix and (trigger.subeventPrefix:find("SPELL") or trigger.subeventPrefix == "RANGE" or trigger.subeventPrefix:find("DAMAGE"))
         end,
-        hidden = WeakAuras.IsClassic(),
+        test = WeakAuras.IsClassic() and "GetSpellInfo(%q) == spellName" or "%d == spellId",
```

Comment:
you can probably do test =  WeakAuras.IsClassic() and "GetSpellInfo(%q) == spellName" or nil

Reply:
i force pushed it